bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/handler/BondDeviceHandler.java

```diff
@@ -205,6 +205,27 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 }
                 break;
 
+            case CHANNEL_RAW_FAN_SPEED:
+                logger.trace("Fan raw speed command");
+                if (command instanceof DecimalType) {
+                    value = ((DecimalType) command).intValue();
+                    BondDeviceProperties devProperties = this.deviceProperties;
+                    if (devProperties != null) {
+                        if (value == 0) {
+                            action = BondDeviceAction.TURN_OFF;
+                            value = null;
+                        } else {
+                            action = BondDeviceAction.SET_SPEED;
+                            value = Math.min(value, devProperties.maxSpeed);
+                        }
+                        logger.trace("Fan raw speed command with speed set as {}", value);
```

Comment:
maybe log the action as well as the value, since it might be TURN_OFF?

Reply:
it would show up as null there, but i can move that up, I agree it makes more logical sense to talk about off vs speed.